extensions/omni_sql/src/execute.sql

```diff
@@ -27,4 +27,7 @@ begin
     return;
 end;
 $pgsql$
-    language plpgsql;
\ No newline at end of file
+    language plpgsql;
+
+
+comment on function omni_sql.execute is 'Runs an arbitrary SQL (possibly prepared) statement. If the statement is a prepared statement, one can define more precise types for the statement using the "types" parameter.'
```

Comment:
I like this idea, but it is important to note that none of the functions across the platform have this. This can be a start.

Reply:
I am also not sure whether we should talk about prepared statements here as it may be confused with [`PREPARE`](https://www.postgresql.org/docs/current/sql-prepare.html)

Reply:
let me retry the text.

Reply:
> I am also not sure whether we should talk about prepared statements here as it may be confused with [`PREPARE`](https://www.postgresql.org/docs/current/sql-prepare.html)

I was thinking about that. Since we are expecting the end user to heavily interact with the DBMS, accessing the docstrings live would be a help, imho.

Reply:
I agree that this can be of help indeed. The endpoint itself can be used to query the docstrings. Was just pointing out that the rest of the extensions don't have this (yet)